hack/mkdocs-generate-conformance.py

```diff
@@ -50,28 +54,58 @@ def on_pre_build(config, **kwargs):
                                                 'HTTPRoutePathRedirect', 'HTTPRouteHostRewrite', 'HTTPRoutePathRewrite', 'HTTPRouteRequestMirror', 'HTTPRouteRequestMultipleMirrors', 'HTTPRouteRequestTimeout', 'HTTPRouteBackendTimeout', 'HTTPRouteParentRefPort']
 
 
-def generate_conformance_tables(reports):
+# NOTE add past versions of implementations to here when new release is cut
+past_versions = """
+??? "Previous Gateway API Versions"
 
-    gateway_http_table = generate_profiles_report(reports, 'HTTP')
-    gateway_http_table = gateway_http_table.rename_axis('Organization')
 
-    # Currently no implementation has extended supported features listed.
-    # Can uncomment once a list is needed to keep track
-    # gateway_tls_table = generate_profiles_report(reprots,'TLS')
+    The latest release of Gateway API is shown here. Past Gateway API implementations and feature support can be found here:
+
+    - [v1.0.0](./implementation-table-v1.0.0.md)
```

Comment:
Thinking about this more, maybe we should aim for the following in terms of file structure:

1. `/site-src/implementations/{VERSION}.md`
2. Don't track patch versions, all reports for a minor version get lumped together
3. In published site, we end up with https://gateway-api.sigs.k8s.io/implementations/v1.1

Reply:
This approach would mean that we wouldn't publish _any_ reports for a release until we met the min criteria (one month or three implementations), but I think it might be simplest to keep track of. We could also choose to limit this to something like the 3 most recent Gateway API versions so the list doesn't become too long.